conduit/data/block_export_data.go

```diff
@@ -16,6 +18,7 @@ type InitProvider interface {
 	GetGenesis() *sdk.Genesis
 	SetGenesis(*sdk.Genesis)
 	NextDBRound() sdk.Round
+	GetTelemetryClient() telemetry.Client
```

Comment:
The other nice thing about this approach is that you can create a test client for unit testing the event.